package.json

```diff
@@ -81,7 +82,6 @@
   "dependencies": {
     "blakejs": "1.2.1",
     "detect-gpu": "^5.0.5",
-    "env": "^0.0.2",
```

Comment:
finally getting rid of this stupid dependency

Reply:
Hey, I also want to contribute to this project, Any pre-requisites, I have decent experience with Solidity and Javascript

Reply:
Yeah sure, JS experience is enough to start contributing! Maybe you find something you can do labeled with `good first issue`.
But if you notice something else that should be improved, were always happy to take suggestions! Also, what snarkyjs needs are more community-maintained libraries

Reply:
thank you for your interest @0xvicky 
see https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md